src/Services/GroupService.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+/**
+ * GroupService.php
+ * @package openemr
+ * @link      http://www.open-emr.org
+ * @author    Stephen Nielson <stephen@nielson.org>
+ * @copyright Copyright (c) 2021 Stephen Nielson <stephen@nielson.org>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Services;
+
+use OpenEMR\Common\Database\QueryUtils;
+use OpenEMR\Common\Uuid\UuidMapping;
+use OpenEMR\Common\Uuid\UuidRegistry;
+use OpenEMR\Services\Search\FhirSearchWhereClauseBuilder;
+use OpenEMR\Validators\ProcessingResult;
+
+class GroupService extends BaseService
+{
+    const PRACTITIONER_TABLE = "users";
+
+    /**
+     * Default constructor.
+     */
+    public function __construct()
+    {
+        UuidRegistry::createMissingUuidsForTables([self::PRACTITIONER_TABLE]);
+        UuidMapping::createMissingResourceUuids("Group", self::PRACTITIONER_TABLE);
```

Comment:
Need to add this new item to UUID_MAPPING_DEFINITIONS array in src/Common/Uuid/UuidMapping.php
(so it will be autocreated by the uuid background service and during upgrades)